packages/sui-studio/scripts/postinstall.js

```diff
@@ -0,0 +1,81 @@
+const fse = require('fs-extra')
+const fg = require('fast-glob')
+
+console.log(`Looking for test folder in ${process.env.INIT_CWD}`)
+
+if (fse.existsSync(`${process.env.INIT_CWD}/test`)) {
+  console.log('Test folder in place')
+  process.exit(0)
+}
+console.log(`Generating test folder in ${process.env.INIT_CWD}`)
+
+const components = fg
+  .sync([
+    `${process.env.INIT_CWD}/components/**/src/index.js`,
+    '!**/node_modules/**'
+  ])
+  .map(path =>
+    path
+      .replace(process.env.INIT_CWD + '/components/', '')
+      .replace('/src/index.js', '')
+  )
+
+const showError = msg => {
+  console.error(msg)
+  process.exit(1)
+}
+
+const writeFile = async (path, body) => {
+  return fse
+    .outputFile(path, body)
+    .then(() => {
+      console.log(`Modified ${path}`)
+    })
+    .catch(() => {
+      showError(`Fail modifying ${path}`)
+    })
+}
+
+const createDir = path => {
+  return fse
+    .mkdirp(path)
+    .then(() => {
+      console.log(`Created ${path}`)
+    })
+    .catch(() => {
+      showError(`Fail creating ${path}`)
+    })
+}
+
+const BODY_TEST = `/**
+ * Remember: YOUR COMPONENT IS DEFINE GLOBALLY
+ * */
+
+/* eslint react/jsx-no-undef:0 */
+
+// import React from 'react'
+// import {render} from '@testing-library/react'
+
+import chai, {expect} from 'chai'
+import chaiDOM from 'chai-dom'
+
+chai.use(chaiDOM)
+
+describe('AtomButton', () => {
```

Comment:
Can we at least create the component name with the category and name of the component so we don't create always AtomButton?